src/classes/SchemaServiceClient.js

```diff
@@ -24,69 +24,191 @@ class SchemaServiceClient {
 		this.apiKey = apiKey;
 	}
 
-	async getTenant(tenantId, organizationCode) {
-		logger.info(`OrgCode - getTenant: ${organizationCode}`);
+	async getMesh(organizationCode, projectId, workspaceId, meshId) {
 		const config = {
 			method: 'get',
-			url: `${this.schemaManagementServiceUrl}/api-admin/organizations/${organizationCode}/tenants/${tenantId}?api_key=${this.apiKey}`,
+			url: `${this.schemaManagementServiceUrl}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}?api_key=${this.apiKey}`,
```

Comment:
why did we decide to remove the api-admin here ? config.json currently looks like this:
```{
    "baseUrl": "https://commerce-beta.adobe.io",       ---> just the baseUrl without the /api-admin
    "apiKey": "commerce-graphql-onboarding"
}
```
Did we decide to change the config.json to have  baseUrl with  /api-admin appended ? If so, we will have to change [here as well](https://github.com/adobe/aio-cli-plugin-commerce-admin/pull/12/files#diff-49ba0c85360693b8e89ae0e0428284a55db749f08f2b9993d26910def646e02fL42)

Reply:
Ideally `/api-admin` will not be part of every SMS server. It just happens to be part of our current URLs. So lets not hard code it into the code. `baseUrl` should be SMS URL and every other path that we hard code in the code like getting `mesh` etc should still work.